confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/internal/input/DefaultConfluenceInputContext.java

```diff
@@ -32,6 +33,8 @@ public class DefaultConfluenceInputContext implements ConfluenceInputContext
 {
     private ThreadLocal<ConfluenceInputProperties> properties = new ThreadLocal<>();
 
+    private ThreadLocal<ConfluenceXMLPackage> xmlPackage = new ThreadLocal<>();
```

Comment:
Would be better for consistency to have a single ThreadLocal which contains an entry with both the ConfluenceInputProperties and the ConfluenceXMLPackage. That means the #set should take both too.